install/playground/docker/nginx.conf

```diff
@@ -1,3 +1,10 @@
+server {
+	listen 80;
+	listen [::]:80;
+	server_name playground.meshery.io;
+	return 301 https://playground.meshery.io$request_uri;
```

Comment:
```suggestion
        return 301 https://$host$request_uri;
```

Reply:
Let's not hardcode the hostname. 
Since we are defining the server_name with `playground.meshery.io`, where `playground.meshery.io` is the hostname to which our nginx server's port 80 is listening to.

So, $host would be `playground.meshery.io` in this case.